### ISSUE TITLE
A bunch of Akaviri Deserters spawn on top of you near the Standing stones around riverwood (Thief,Warrior,Mage))

### ISSUE THREAD

**Issue by cheesemanhugs** [Althro/Tinvaak#9]
If coming at the stones from the path intended these Akaviri Deserters will spawn right on top of you. Basically instant death if low level.

**Comment by Althro** [comment]
I'll need more information to go on to look into this. I've not experienced this before. Some details I'll need from you:

- Where did you start in Game.
- What is your current level.
- What direction did you approach the standing stones from. From Riverwood up the path past Embershard, or down from Helgen.
- What level were the deserters.

Once I have that, I can look into it some more.

**Comment by cheesemanhugs** [comment]
I started my playthrough at the border near falkreath. Took a carriage to riverwood then took out a bandit camp or bounty behind the standing stones. On my way back to riverwood I went to the standing stones and got the issue. I kind of fell on top of them as I decided to jump off the side of the little mountain rather than take the intended path. When taking the intended path that's when they spawn **on top** of you. I  came at it from the falkreath watchtower more like on my first encounter (So I was walking towards riverwood and helgen not from them). These pictures might help. The only difference with this path and the intended path is that they don't spawn on top of you when going this way. Rather they spawn in front of you.
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 01 - 16 03 22 05](https://user-images.githubusercontent.com/64744940/116793840-315e9d80-aa97-11eb-9fbd-764aa1dc1646.png)
![The Elder Scrolls V  Skyrim Special Edition Screenshot 2021 05 01 - 16 03 26 24](https://user-images.githubusercontent.com/64744940/116793844-39b6d880-aa97-11eb-81b2-4fdc0a44b415.png)

-I was a lvl 2 Redguard
-The deserters (they comprised of bandits, bandit vampire fledling, poacher, stray companion welp) (4 guys) were lvl 1 and lvl 2 at the highest. 
-(BIG THING) I was trying to replicate this while typing this and couldn't :(. That was until I remembered what time of day this happened. It was around evening so I waited until 6pm and then the deserters spawned consistently.
